code_challenges.rb

```diff
@@ -33,19 +45,60 @@
 album3 = 'Abbey Road'
 # Expected output: 'bby Rd'
 
-
-
+# create method takes a string as an argument
+def no_vowels string 
+# removes all vowels from string
+        # string.downcase.delete "aeiou"
+        string.delete "aeiouAEIOU"
+end
+p no_vowels(album1)
+p no_vowels(album2)
+p no_vowels(album3)
 
 
 # --------------------4) Create a class called Bike that is initialized with a model, wheels, and current_speed. The default number of wheels is 2. The current_speed should start at 0. Create a get_info method that returns a sentence with all the data from the bike object.
 
 # Expected output example: 'The Trek bike has 2 wheels and is going 0 mph.'
 
-
-
-
+# create a class called Bike
+class Bike
+    # create initialize method
+    def initialize(model, wheels, current_speed)
```

Comment:
looks like the initialize method only needs 1 parameter because the other two (wheels and current_speed) are not being using in the logic within the method